### ISSUE TITLE
timear forsees infinite agents

### ISSUE THREAD

**Issue by Aladeus** [dalaranwow/dalaran-wow#5558]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Went to Culling of Stratholme heroic dungeon with Timear Forsees Infinite Agents quest, only got credit for 3/4 agents.
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by alexuktc** [comment]
This is a server wide issue not limited to what you have experienced. I am not sure if I can call it an issue if it is an intended feature though. 

Assuming there is an npc that does damage to a mob you're attacking you may not get credit for killing it (thus no loot, experience and quest progression) if the NPC has done >50% of the damage on the mob. This was incredibly annoying during ICC quests where you had NPCs doing very high damage and you had to do more than half of the enemy's hp to get credit. 

While this is the only server where I have encountered this it may be an intended feature and I can't speak for or against it.

**Comment by Golganeth** [comment]
General server wide mechanic, players have to deal most of the damage in order to get the kill credit.
Closing.
